releaseNotes/AzureDevOpsPipelineExtension.md

```diff
@@ -2,6 +2,11 @@
 
 The Azure App Configuration extension in Azure DevOps pipeline has been published to [Azure DevOps marketplace](https://marketplace.visualstudio.com/items?itemName=AzureAppConfiguration.azure-app-configuration-task&ssr=false#overview). The version of the Azure App Configuration task being used will print to the console when the task is executed: ![sample](pictures/AzureDevOpsExtensionVersionSample.PNG)
 
+### v1.6.4 - June, 19 2019
+* Improve [homepage documentation](https://marketplace.visualstudio.com/items?itemName=AzureAppConfiguration.azure-app-configuration-task):
+  * Add the link the the release note.
```

Comment:
Added a link to the release notes.

Reply:
fixed.

Reply:
Double "the"

Reply:
Yeah looks like that change got left out

"the the" -> "to the"

Reply:
Yes, fixed.